internal/ingress/annotations/annotations.go

```diff
@@ -74,7 +74,7 @@ const DeniedKeyName = "Denied"
 type Ingress struct {
 	metav1.ObjectMeta
 	BackendProtocol      string
-	Alias                string
+	Alias                []string
```

Comment:
Would it be too hard to rename this to `Aliases`?

Reply:
done